plugin/src/utils/cache.ts

```diff
@@ -15,30 +15,41 @@ import {
 } from "~/steps/create-schema-customization/helpers"
 
 import { createMediaItemNode } from "~/steps/source-nodes/fetch-nodes/fetch-referenced-media-items"
+import { Node } from "gatsby"
 
 const MAX_CACHE_SIZE = 250
 const TTL = Number.MAX_SAFE_INTEGER
 const cacheDir = `.wordpress-cache/caches`
 
+type Store = manager.StoreConfig["store"]
+
+interface CacheOptions {
+  name?: string
+  store?: Store
+}
 export default class Cache {
-  constructor({ name = `db`, store = fsStore } = {}) {
-    this.name = name
-    this.store = store
+  private store: Store
+  private name: string
+  private cacheDirectory: string
+  private cache: manager.MultiCache
+  constructor(options?: CacheOptions) {
+    this.name = options?.name || `db`
+    this.store = options?.store || fsStore
```

Comment:
@TylerBarnes this is one of the few changes to execution code left. pretty significant change. i had trouble recreating this default arg pattern in typescript for some reason but I can try to eliminate execution changes here entirely to see if that helps. it would make sense if this is it, since it deals with the hard fs cache

Reply:
ah, this is why it didn't work before.

before I'd tried this
![image](https://user-images.githubusercontent.com/1368727/102498131-91ecc100-4047-11eb-9a11-51ea4174cd43.png)

so instead, i tried this ✅ 
![image](https://user-images.githubusercontent.com/1368727/102498231-b21c8000-4047-11eb-9dd4-9023c3724da3.png)